dataikuapi/dss/project.py

```diff
@@ -249,6 +249,29 @@ def set_permissions(self, permissions):
         return self.client._perform_empty(
             "PUT", "/projects/%s/permissions" % self.project_key, body = permissions)
 
+    def get_interest(self):
+        """
+        Get the interest of this project. The interest means the number of watchers and the number of star.
+
+        :return: a dict object containing the project's interest
+        :rtype: dict
+        """
+        return self.client._perform_json("GET","/projects/%s/interest" % self.project_key)
+
+    def get_timeline(self, itemCount=100):
+        """
+        Get a timeline of this project. The timeline consists of information about the creation of this project
+        (by whom, and when), the last modification of this project (by whom and when), a list of contributors,
+        and a list of modification. This list contains "itemCount" number of element (default: 100). If itemCount
```

Comment:
...and a list of modification**s**.

Reply:
>This list contains "itemCount" number of element (default: 100). If itemCount....

This list of modifications contains a maximum of itemCount elements (default: 100).

Reply:
Also might be a good idea to describe the various top-level fields in the dict (createdOn, ...)

Reply:
Should I describe them, or simply list them